ssl/ssl_lib.c

```diff
@@ -2506,6 +2506,26 @@ STACK_OF(SSL_CIPHER) *SSL_CTX_get_ciphers(const SSL_CTX *ctx)
     return NULL;
 }
 
+/*
+ * Distinguish between ciphers controlled by set_ciphersuite() and
+ * set_cipher_list() when counting.
+ */
+static int cipher_tlist_ls12_num(STACK_OF(SSL_CIPHER) *sk)
```

Comment:
Typo: cipher_tlist_*ls12*_num  -> tls ?

Reply:
@tmshort did you mean `cipher_list_tls12_num()` in https://github.com/openssl/openssl/pull/7759#discussion_r238824475 ?

Reply:
@sam-github : yes, or I miss the meaning of this method name ?
Or is there some name collision ?

Reply:
Yes that is what @tmshort meant.